tests/test_odp_event_manager.py

```diff
@@ -392,15 +393,48 @@ def test_odp_event_manager_flush_timeout(self, *args):
             event_manager.api_manager, 'send_odp_events', new_callable=CopyingMock, return_value=False
         ) as mock_send:
             event_manager.send_event(**self.events[0])
+            mock_send.assert_not_called()
             event_manager.send_event(**self.events[1])
             event_manager.event_queue.join()
-            time.sleep(1)
+            time.sleep(1)   # ensures that the flush interval time has passed
 
         mock_logger.error.assert_not_called()
         mock_logger.debug.assert_any_call('ODP event queue: flushing on interval.')
         mock_send.assert_called_once_with(self.api_key, self.api_host, self.processed_events)
         event_manager.stop()
 
+    # TODO - new test
+    def test_odp_event_manager_flush_interval_is_zero(self, *args):
```

Comment:
Let's chat offline about this test- we need to add some additional joins to make this more robust. You can see that it failed the pypy run.